Objective-C/CBLReplicator.mm

```diff
@@ -565,9 +565,12 @@ - (void) onDocsEnded: (NSArray<CBLReplicatedDocument*>*)docs pushing: (BOOL)push
                                                           error: &error]) {
                     [doc resetError]; // Reset the error as successfully resolving the conflicts
```

Comment:
Reset the error here has no point as the ReplicatedDocument has already been notified with the conflict error.

Reply:
i am not changing this for now, will do this handling in separate PR.